src/script/common/c_internal.cpp

```diff
@@ -155,24 +155,28 @@ static void script_log(lua_State *L, const std::string &message,
 		infostream << script_get_backtrace(L) << std::endl;
 }
 
-void log_deprecated(lua_State *L, const std::string &message, int stack_depth)
-{
+DeprecatedHandlingMode getDeprecatedLuaAPIHandlingMode() {
```

Comment:
brace needs to be on following line
also global functions need to be snake case

Reply:
Our code style is so incredibly arbitrary